pkg/fqdn/dnsproxy/proxy.go

```diff
@@ -71,12 +71,12 @@ type DNSProxy struct {
 	// BindPort is the port in BindAddr.
 	BindPort uint16
 
-	// LookupEndpointIDByIP is a provided callback that returns the endpoint ID
+	// LookupNonrestoredEP is a provided callback that returns the endpoint ID
 	// as a uint16.
 	// Note: this is a little pointless since this proxy is in-process but it is
 	// intended to allow us to switch to an external proxy process by forcing the
 	// design now.
-	LookupEndpointIDByIP LookupEndpointIDByIPFunc
+	LookupNonrestoredEP LookupEndpointIDByIPFunc
```

Comment:
nit: Could maybe refer to these via `LookupRegisteredEndpoint()` since it should be tied into endpointManager registration?

Reply:
Fixed.